packages/flutter_markdown/lib/src/_functions_io.dart

```diff
@@ -24,23 +24,62 @@ final ImageBuilder kDefaultImageBuilder = (
   double? height,
 ) {
   if (uri.scheme == 'http' || uri.scheme == 'https') {
-    return Image.network(uri.toString(), width: width, height: height);
+    return Image.network(
+      uri.toString(),
+      width: width,
+      height: height,
+      errorBuilder: kDefaultImageErrorWidgetBuilder,
+    );
   } else if (uri.scheme == 'data') {
     return _handleDataSchemeUri(uri, width, height);
   } else if (uri.scheme == 'resource') {
-    return Image.asset(uri.path, width: width, height: height);
+    return Image.asset(
+      uri.path,
+      width: width,
+      height: height,
+      errorBuilder: kDefaultImageErrorWidgetBuilder,
+    );
   } else {
     final Uri fileUri = imageDirectory != null
         ? Uri.parse(imageDirectory + uri.toString())
         : uri;
     if (fileUri.scheme == 'http' || fileUri.scheme == 'https') {
-      return Image.network(fileUri.toString(), width: width, height: height);
+      return Image.network(
+        fileUri.toString(),
+        width: width,
+        height: height,
+        errorBuilder: kDefaultImageErrorWidgetBuilder,
+      );
     } else {
-      return Image.file(File.fromUri(fileUri), width: width, height: height);
+      try {
+        return Image.file(
+          File.fromUri(fileUri),
```

Comment:
Unfortunately `File.fromUri` throws, so the `errorBuilder` is not invoked in that case

Reply:
I think this can be fixed by setting the value to a variable then returning it. I'm not 100% certain on that though.

Reply:
I'm not sure if that would work? Per the docs in https://api.dart.dev/dart-io/File/File.fromUri.html

`If uri cannot reference a file this throws [UnsupportedError](https://api.dart.dev/dart-core/UnsupportedError-class.html)`

This aligns with the second error reported in https://github.com/flutter/flutter/issues/158428

`Unsupported operation: Cannot extract a file path from a ttps URI`

So that's why I added a try/catch here, since the Image.file() constructor won't be able to invoke its `errorBuilder`.